### PR TITLE
Highlight translatable fields

### LINKED ISSUE #30: RFC: Highlight translatable fields
I really like that qtranslate-x supports translating custom (meta) fields. However, it is not always clear, which fields are translatable, and which are not. 
This could be improved upon by visually highlighting the translatable field. For example with a small flag icon in at the top right of each field. Like this maybe (option 1):

image

There are probably better ideas for highlighting the fields, but this is just a first idea. What do you think?

### PR DESCRIPTION
**Pull request by mweimerskirch**
A quick technical preview of how the translatable fields could be highlighted. I don't like the layout yet, but feedback is already welcome. 
Fixes #30

### CODE REVIEW

File: admin/js/common.js [hunk 1]
```diff
@@ -562,13 +565,13 @@ var qTranslateX=function(pg)
 								}
 							}
 						}else{
-							continue;
+							continue;
```
Comment (johnclause) [inline admin/js/common.js]: Hi Michel, I am just curious, what does cause git to show lines like these to be different? Even spacing seems to be all tabs. It is a bit confusing, but not a big problem, I just would like to understand. Do you know what is different in them?

Comment (mweimerskirch) [inline admin/js/common.js]: I think they're Windows line breaks mixed in between the Linux line breaks of the rest of the code. I was unable to remove them from the commit even though I tried with 2 clients (GUI and command line). They were always changed back at commit time.

Comment (johnclause) [inline admin/js/common.js]: git should convert line breaks correctly, if you checkout the whole file with git and then put it back with git too? Interesting, it is not too important, but there must be a way to deal with it ...

Were mixed line breaks there to begin with? I may need to check if there are mixed linebreaks in all the files. I thought PHP would issue warning on this, would not it?

Comment (mweimerskirch) [inline admin/js/common.js]: I found this: https://help.github.com/articles/dealing-with-line-endings/#platform-all
My system wide Git configuration was set to "autocrlf = input" which is why it converted the linebreaks.

Comment (johnclause) [inline admin/js/common.js]: Wow, I am glad you figured that out :+1:  We never stop learning ...

**Pull request merged**